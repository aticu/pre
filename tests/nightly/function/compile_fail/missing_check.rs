use pre::pre;

#[pre("is bar")]
fn foo() {}

fn main() {
    foo()
}
