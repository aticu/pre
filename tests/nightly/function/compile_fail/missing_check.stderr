error[E0061]: this function takes 1 argument but 0 arguments were supplied
 --> $DIR/missing_check.rs:7:5
  |
3 |   #[pre("is bar")]
  |  _______-
4 | | fn foo() {}
  | |___________- defined here
...
7 |       foo()
  |       ^^^-- supplied 0 arguments
  |       |
  |       expected 1 argument
